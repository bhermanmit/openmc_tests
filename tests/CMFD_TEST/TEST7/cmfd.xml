<?xml version="1.0"?>
<cmfd>

  <mesh>
    <origin> -15 -1 -1 </origin>
    <dimension> 30 1 1 </dimension>
    <width> 1.0 2.0 2.0 </width>
    <albedo> 0.0 0.0 1.0 1.0 1.0 1.0 </albedo>
    <energy> 0.0 0.625e-6 1.0e-3 1.0 10.0 20.0 </energy> 
  </mesh>

</cmfd>
