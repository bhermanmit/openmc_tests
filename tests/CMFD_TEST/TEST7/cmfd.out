MESH (1. 1, 1)  GROUP 1           1.08000E-05
MESH (1. 1, 1)  GROUP 2           3.25840E-03
MESH (1. 1, 1)  GROUP 3           -3.35600E-04
MESH (1. 1, 1)  GROUP 4           -1.59850E-03
MESH (1. 1, 1)  GROUP 5           -1.33510E-03
MESH (2. 1, 1)  GROUP 1           7.10000E-06
MESH (2. 1, 1)  GROUP 2           4.71900E-04
MESH (2. 1, 1)  GROUP 3           1.88500E-04
MESH (2. 1, 1)  GROUP 4           -3.92400E-04
MESH (2. 1, 1)  GROUP 5           -2.75100E-04
MESH (3. 1, 1)  GROUP 1           4.00000E-06
MESH (3. 1, 1)  GROUP 2           -2.40400E-04
MESH (3. 1, 1)  GROUP 3           7.29200E-04
MESH (3. 1, 1)  GROUP 4           -3.35900E-04
MESH (3. 1, 1)  GROUP 5           -1.56900E-04
MESH (4. 1, 1)  GROUP 1           3.60000E-06
MESH (4. 1, 1)  GROUP 2           -2.10000E-05
MESH (4. 1, 1)  GROUP 3           8.57300E-04
MESH (4. 1, 1)  GROUP 4           -6.41500E-04
MESH (4. 1, 1)  GROUP 5           -1.98400E-04
MESH (5. 1, 1)  GROUP 1           6.30000E-06
MESH (5. 1, 1)  GROUP 2           3.17840E-03
MESH (5. 1, 1)  GROUP 3           -4.91100E-04
MESH (5. 1, 1)  GROUP 4           -2.57080E-03
MESH (5. 1, 1)  GROUP 5           -1.22800E-04
MESH (6. 1, 1)  GROUP 1           5.70699E-08
MESH (6. 1, 1)  GROUP 2           -2.19493E-03
MESH (6. 1, 1)  GROUP 3           5.90000E-04
MESH (6. 1, 1)  GROUP 4           1.28160E-03
MESH (6. 1, 1)  GROUP 5           3.12100E-04
MESH (7. 1, 1)  GROUP 1           4.23156E-06
MESH (7. 1, 1)  GROUP 2           2.41093E-04
MESH (7. 1, 1)  GROUP 3           -8.80342E-04
MESH (7. 1, 1)  GROUP 4           6.05701E-04
MESH (7. 1, 1)  GROUP 5           1.81000E-05
MESH (8. 1, 1)  GROUP 1           6.02304E-06
MESH (8. 1, 1)  GROUP 2           7.61678E-04
MESH (8. 1, 1)  GROUP 3           -1.11057E-03
MESH (8. 1, 1)  GROUP 4           3.33301E-04
MESH (8. 1, 1)  GROUP 5           2.20000E-06
MESH (9. 1, 1)  GROUP 1           2.82653E-06
MESH (9. 1, 1)  GROUP 2           9.97897E-04
MESH (9. 1, 1)  GROUP 3           -1.20288E-03
MESH (9. 1, 1)  GROUP 4           1.98900E-04
MESH (9. 1, 1)  GROUP 5           1.50000E-06
MESH (10. 1, 1)  GROUP 1          2.72537E-06
MESH (10. 1, 1)  GROUP 2          1.11943E-03
MESH (10. 1, 1)  GROUP 3          -1.24279E-03
MESH (10. 1, 1)  GROUP 4          1.19800E-04
MESH (10. 1, 1)  GROUP 5          9.00000E-07
MESH (11. 1, 1)  GROUP 1          1.01945E-06
MESH (11. 1, 1)  GROUP 2          1.03071E-03
MESH (11. 1, 1)  GROUP 3          -1.11796E-03
MESH (11. 1, 1)  GROUP 4          7.59001E-05
MESH (11. 1, 1)  GROUP 5          5.00000E-07
MESH (12. 1, 1)  GROUP 1          2.62392E-06
MESH (12. 1, 1)  GROUP 2          1.09258E-03
MESH (12. 1, 1)  GROUP 3          -1.16220E-03
MESH (12. 1, 1)  GROUP 4          6.66010E-05
MESH (12. 1, 1)  GROUP 5          -1.00000E-07
MESH (13. 1, 1)  GROUP 1          1.72802E-06
MESH (13. 1, 1)  GROUP 2          1.07541E-03
MESH (13. 1, 1)  GROUP 3          -1.10225E-03
MESH (13. 1, 1)  GROUP 4          3.32010E-05
MESH (13. 1, 1)  GROUP 5          2.00000E-07
MESH (14. 1, 1)  GROUP 1          1.92608E-06
MESH (14. 1, 1)  GROUP 2          1.16180E-03
MESH (14. 1, 1)  GROUP 3          -1.18069E-03
MESH (14. 1, 1)  GROUP 4          9.50010E-06
MESH (14. 1, 1)  GROUP 5          2.00000E-07
MESH (15. 1, 1)  GROUP 1          6.62398E-06
MESH (15. 1, 1)  GROUP 2          1.07336E-03
MESH (15. 1, 1)  GROUP 3          -1.11164E-03
MESH (15. 1, 1)  GROUP 4          3.22002E-05
MESH (15. 1, 1)  GROUP 5          -2.00000E-07
MESH (16. 1, 1)  GROUP 1          3.92214E-06
MESH (16. 1, 1)  GROUP 2          1.02614E-03
MESH (16. 1, 1)  GROUP 3          -1.06458E-03
MESH (16. 1, 1)  GROUP 4          2.87001E-05
MESH (16. 1, 1)  GROUP 5          4.00000E-07
MESH (17. 1, 1)  GROUP 1          6.22286E-06
MESH (17. 1, 1)  GROUP 2          1.13756E-03
MESH (17. 1, 1)  GROUP 3          -1.19244E-03
MESH (17. 1, 1)  GROUP 4          4.01002E-05
MESH (17. 1, 1)  GROUP 5          8.00000E-07
MESH (18. 1, 1)  GROUP 1          2.23790E-07
MESH (18. 1, 1)  GROUP 2          1.04882E-03
MESH (18. 1, 1)  GROUP 3          -1.07420E-03
MESH (18. 1, 1)  GROUP 4          2.89001E-05
MESH (18. 1, 1)  GROUP 5          -4.00000E-07
MESH (19. 1, 1)  GROUP 1          1.42668E-06
MESH (19. 1, 1)  GROUP 2          1.07223E-03
MESH (19. 1, 1)  GROUP 3          -1.12239E-03
MESH (19. 1, 1)  GROUP 4          6.15001E-05
MESH (19. 1, 1)  GROUP 5          -1.00000E-06
MESH (20. 1, 1)  GROUP 1          6.52234E-06
MESH (20. 1, 1)  GROUP 2          9.81277E-04
MESH (20. 1, 1)  GROUP 3          -1.05735E-03
MESH (20. 1, 1)  GROUP 4          6.58004E-05
MESH (20. 1, 1)  GROUP 5          6.00000E-07
MESH (21. 1, 1)  GROUP 1          4.27180E-07
MESH (21. 1, 1)  GROUP 2          1.09218E-03
MESH (21. 1, 1)  GROUP 3          -1.21862E-03
MESH (21. 1, 1)  GROUP 4          1.36500E-04
MESH (21. 1, 1)  GROUP 5          1.50000E-06
MESH (22. 1, 1)  GROUP 1          4.72321E-06
MESH (22. 1, 1)  GROUP 2          9.67004E-04
MESH (22. 1, 1)  GROUP 3          -1.16254E-03
MESH (22. 1, 1)  GROUP 4          1.95400E-04
MESH (22. 1, 1)  GROUP 5          1.00000E-06
MESH (23. 1, 1)  GROUP 1          2.12273E-06
MESH (23. 1, 1)  GROUP 2          8.29848E-04
MESH (23. 1, 1)  GROUP 3          -1.18372E-03
MESH (23. 1, 1)  GROUP 4          3.45500E-04
MESH (23. 1, 1)  GROUP 5          3.10000E-06
MESH (24. 1, 1)  GROUP 1          2.63000E-06
MESH (24. 1, 1)  GROUP 2          2.28760E-04
MESH (24. 1, 1)  GROUP 3          -8.28205E-04
MESH (24. 1, 1)  GROUP 4          5.93800E-04
MESH (24. 1, 1)  GROUP 5          2.06000E-05
MESH (25. 1, 1)  GROUP 1          -4.24509E-06
MESH (25. 1, 1)  GROUP 2          -2.21527E-03
MESH (25. 1, 1)  GROUP 3          6.61538E-04
MESH (25. 1, 1)  GROUP 4          1.24680E-03
MESH (25. 1, 1)  GROUP 5          3.20800E-04
MESH (26. 1, 1)  GROUP 1          1.18000E-05
MESH (26. 1, 1)  GROUP 2          3.24040E-03
MESH (26. 1, 1)  GROUP 3          -5.70500E-04
MESH (26. 1, 1)  GROUP 4          -2.58870E-03
MESH (26. 1, 1)  GROUP 5          -9.30000E-05
MESH (27. 1, 1)  GROUP 1          3.20000E-06
MESH (27. 1, 1)  GROUP 2          -2.14000E-05
MESH (27. 1, 1)  GROUP 3          8.36700E-04
MESH (27. 1, 1)  GROUP 4          -6.09600E-04
MESH (27. 1, 1)  GROUP 5          -2.08900E-04
MESH (28. 1, 1)  GROUP 1          7.60000E-06
MESH (28. 1, 1)  GROUP 2          -2.75700E-04
MESH (28. 1, 1)  GROUP 3          7.54100E-04
MESH (28. 1, 1)  GROUP 4          -3.13900E-04
MESH (28. 1, 1)  GROUP 5          -1.72100E-04
MESH (29. 1, 1)  GROUP 1          5.10000E-06
MESH (29. 1, 1)  GROUP 2          4.34300E-04
MESH (29. 1, 1)  GROUP 3          2.61500E-04
MESH (29. 1, 1)  GROUP 4          -4.09700E-04
MESH (29. 1, 1)  GROUP 5          -2.91200E-04
MESH (30. 1, 1)  GROUP 1          8.90000E-06
MESH (30. 1, 1)  GROUP 2          3.26760E-03
MESH (30. 1, 1)  GROUP 3          -3.61100E-04
MESH (30. 1, 1)  GROUP 4          -1.60220E-03
MESH (30. 1, 1)  GROUP 5          -1.31320E-03
